### ISSUE TITLE
Recently uploaded labels do not meet color contrast accessibility requirements

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6746]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.0), the Keywords and Depositor labels on the Recently uploaded tab do not meet WCAG AA minimum color contrast requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to dev.nurax.samvera.org and click on the `Recently Uploaded` link
3. Run the Site Improve browser extension

![color_contrast_minimum_keyword](https://github.com/samvera/hyrax/assets/32644254/a71d7ef1-fbad-4159-905f-ec5d01b74ffb)

### Rationale

Hyrax (and nurax) should be accessible to users out of the box.

**Comment by rkuehn-uofl** [comment]
Recommend changing .recent-label from #6c757d to #434C54 (~/app/assets/stylesheets/hyrax/_recent.scss - line 24)

**Comment by rkuehn-uofl** [comment]
Fixed by #6935

**Comment by jlhardes** [comment]
PR for #6745 does not impact the Depositor and Keywords labels (or does not impact them enough) to resolve the SiteImprove errors. This issue requires something else to increase the color contrast.

**Comment by jlhardes** [comment]
It looks like adding the class "text-dark" after "recent-label" on the `<span>`s for both the depositor and keyword labels in `app/views/hyrax/homepage/_recent_document.html.erb` will make enough contrast to pass the SiteImprove check.

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue at the AA level. There is a note to increase contrast at the AAA level.
